packages/env/lib/build-docker-compose-config.js

```diff
@@ -113,6 +113,34 @@ module.exports = function buildDockerComposeConfig( config ) {
 	const developmentPorts = `\${WP_ENV_PORT:-${ config.env.development.port }}:80`;
 	const testsPorts = `\${WP_ENV_TESTS_PORT:-${ config.env.tests.port }}:80`;
 
+	// Set the WordPress, WP-CLI, PHPUnit PHP version if defined.
+	const phpVersion = config.env.development.phpVersion
```

Comment:
We should let `config.env.tests` control the PHP version used for the tests service

Reply:
So, the addition of the phpVersion wasn't intended to be used at the environment level to have different versions being used in different environments. Partly because there is the option of using an environment variable and I only have 1 environment variable available to use. The whole configuration design requires all of these settings to be at the environment level so I chose here to just use the development environment setting. The fact that there is only development and tests environments I find it hard to see the use case for having them using different PHP versions, just like with the fact that it is the same WordPress version being used between environments. I'll not also that attempting to have a development and test phpVersion setup will further complicate that code later when defining all of the images since the 3 image types that need have different PHP version formats used in the image tags.

Reply:
@noahtallen if you can make a case for differing PHP versions between `development` and `tests` I could be persuaded to take this code farther to accept that condition.

Reply:
You can set different WordPress versions for each instance because you can set every config option differently for each environment. (So you can set the core source to be different things.) I'm not sure what specific use case there is for it, but when we implemented the code which lets you change things in each environment, we just made it generic so that it works for any option. This would be the only option which isn't configurable between services, so for consistency's sake, I believe it makes sense to set the value for the tests instance.

For environment variable, since those are more of an edge case, maybe we just keep the one and let it override things for all environment services. 

To me, we could just add an extra couple of lines for the tests-wp and tests-cli images, and then just use the tests version for the phpunit service.

Reply:
OK, I can get behind the consistency factory. I'll work up those changes and update. Thanks!

Reply:
@noahtallen this should be set now. Thanks!

Reply:
nice, the code looks great to me! I'll test it out locally tomorrow :)